src/Monolog/Handler/SwiftMailerHandler.php

```diff
@@ -79,7 +79,7 @@ protected function buildMessage(string $content, array $records): Swift_Message
         }
 
         $message->setBody($content, $mime);
-        $message->setDate(time());
+        $message->setDate(new DateTimeImmutable());
```

Comment:
This needs a slash beforehand to import from the core namespace. If you run the tests before you commit, you will see:
```
Error: Class 'Monolog\Handler\DateTimeImmutable' not found
```